gapii/cc/spy.cpp

```diff
@@ -172,8 +172,10 @@ Spy::Spy()
 #else
     mConnection = ConnectionStream::listenSocket("127.0.0.1", "9286");
 #endif                                          // TARGET_OS
-    if (mConnection->write("gapii", 5) != 5) {  // handshake string
+    if (mConnection->write("gapii", 6) != 6) {  // handshake string
```

Comment:
`gapii` is only 5 letters and the other side is still only expecting 5 bytes here.